### PR TITLE
(?d) example

### PR DESCRIPTION
**Pull request by uglygus**

### CODE REVIEW

**Comment by AudriusButkevicius** [comment]
The empty directory quote is confusing, as to me it implies that .DS_Store should be empty, though it doesn't need to be.

**Comment by uglygus** [comment]
I was trying to fit all on one line. How  about this: 
    bar/          # synced
    	.tmpfile  # ignored, (?d) allows deletion if the directory if otherwise empty
        baz       # synced

**Comment by uglygus** [comment]
should read:
(?d).tmpfile  # ignored, (?d) allows deletion if the directory if otherwise empty

**Comment by AudriusButkevicius** [comment]
Ignored, will be deleted if gets in the way of parent directory removal

**Comment by AudriusButkevicius** [comment]
Can yoy add an extra space to fix the comment algnment? Also next time to make commits, drop a message as github does not notify about commits to PR's

File: users/ignoring.rst [hunk 1]
```diff
@@ -113,6 +114,7 @@ and an ``.stignore`` file with the contents::
 all files and directories called "foo", ending in a "2" or starting with
 "qu" will be ignored. The end result becomes::
 
+    .DS_Store	  # ignored, will be deleted if gets in the way of parent directory removal
```
Comment (calmh) [inline users/ignoring.rst]: Shouldn't this actually have the `(?d)` prefix then?

**Comment by uglygus** [comment]
how's that

**Comment by AudriusButkevicius** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as 8748339201835d4ca6fc38577d5eb07795a95542. Thanks, @uglygus!

**Pull request closed without merging**